### PR TITLE
Patched incorrect legacy USDC insurance fund address being used

### PR DESCRIPTION
**Pull request by lukecaan**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/drift.js: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'value')
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:142:36
        at Array.forEach (<anonymous>)
        at getTokenAccountBalances (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:142:15)
        at async sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:348:27)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:148:9
        at async Promise.all (index 0)
        at async Promise.all (index 2)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:193:3

File: projects/drift.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ async function tvl(_, _b, _cb, { api, }) {
 
   const legacyVaults = [
     '6W9yiHDCW9EpropkFV8R3rPiL8LVWUHSiys3YeW6AT6S', // legacy usdc vault
-    '2CqkQvYxp9Mq4PqLvAQ1eryYxebUh4Liyn5YMDtXsYci', // legacy usdc insurance fund
+    'Bzjkrm1bFwVXUaV9HTnwxFrPtNso7dnwPQamhqSxtuhZP', // legacy usdc insurance fund
```
Comment (g1nt0ki) [inline projects/drift.js:19]: this looks like invalid address.

I copied your change from here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/6909/files#diff-0ea3688383fd8f783666f4ad6ce9697f539e9a6c75b9e40feeb2a35ff4c7fd4dR44

Comment (lukecaan) [reply projects/drift.js:19]: Omg sorry someone else in my org asked me to make that change and didn't double check it .. will figure out what the problem is

**Comment by llamatester** [comment]
The adapter at projects/drift.js exports TVL: 
        
 
 
    solana                    12.04 M
    
    total                    12.04 M

File: projects/drift.js (hunk 1 above)
Comment (lukecaan) [reply projects/drift.js]: Fixed .. I accidentally added a 'P' on the end 😆 , appreciate your help :)

**Pull request merged**